src/test/java/com/deloitte/elrr/datasync/controller/ImportsControllerTest.java

```diff
@@ -107,7 +108,11 @@ void getImportsAllTest() throws Exception {
    @Test
    void getImportsTest() throws Exception {
        ImportDTO importDTO = new ImportDTO();
-       mockMvc.perform(get("/api/getImports/{id}", 1L))
+      HttpHeaders headers = new HttpHeaders();
```

Comment:
I had to update this junit because the request has to be sent as https with the changes we made